README.md

```diff
@@ -1,22 +1,5 @@
-# Flarum Website
+# Flarum Website Issues
 
-Docs can be found in the [flarum/docs](https://github.com/flarum/docs) repo.
-
-# Get started 
-
-You can edit small changes online using the GitHub editor. For larger edits it is recommended to make a local checkout and test your changes before you commit them. To do that, you need Ruby. Instructions for Linux/Mac:
-
-```
-git clone https://github.com/flarum/flarum.github.io
-cd flarum.github.io
+This repo is strictly for redirecting flarum.github.io to flarum.org. Further it servs as a way for the community to alert us of issues with our flarum.org site.
 
-cat > Gemfile <<EOM
-source 'https://rubygems.org'
-gem 'github-pages', group: :jekyll_plugins
-EOM
-
-bundle install
-bundle exec jekyll serve
-```
-
-Now, you can view your changes through http://localhost:4000. Jekyll will automatically rebuild the site when you save your edits.
+Docs can be found in the [flarum/docs](https://github.com/flarum/docs) repo.
```

Comment:
Docs are stored, and managed, in the ... repo?

Reply:
The URL points to the flarum/docs repo....